### ISSUE TITLE
Placeholder to update control node to ansible-1 from ansible

### ISSUE THREAD

**Issue by termlen0** [network-automation/tower_workshop#9]
In expectation of PR https://github.com/ansible/workshops/pull/891 merging into the main workshops repo, any references to the control node should be updated to `ansible-1` (network reports for instance). 
I plan to update this issue with an associated PR once the cluster PR is merged into devel.
